utils/ssh.py

```diff
@@ -480,6 +480,14 @@ def set_initial_file_end(self):
             fstat = sshtail._sftp_client.stat(self._remote_filename)
             self._remote_file_size = fstat.st_size  # Seed initial size of file
 
+    def raw_output(self):
+        """Returns the output of the log as one (raw) string"""
+        return '\n'.join(self.lines_as_list())
```

Comment:
1. Look at lines_as_list it did exactly [line for line in self] -> this is why I created this.
2. any suggestion for name and generic '\n'?